src/modules/src/supervisor_state_machine.c

```diff
@@ -63,12 +63,6 @@ static const char* const conditionNames[] = {
 };
 static_assert(sizeof(conditionNames) / sizeof(conditionNames[0]) == supervisorCondition_NrOfConditions);
 
-#if SUPERVISOR_TUMBLE_CHECK_ENABLE
```

Comment:
While the solution with the callbacks isn’t the most elegant, it serves its purpose for now.

Reply:
I returned SUPERVISOR_TUMBLE_CHECK_ENABLE to platform_defaults.
Since I would also like to keep the option to enable/disable the detection with a parameter (no need to build and flash the firmware), it is now handled in supervisor.c